sdk/identity/identity/src/client/identityClient.ts

```diff
@@ -21,6 +21,7 @@ import { TokenCredentialOptions } from "../tokenCredentialOptions";
 import {
   TokenResponseParsedBody,
   parseExpiresOn,
+  getRefreshAt,
```

Comment:
Nit, you should move the getRefreshAt above for ESLint ordering purposes

Reply:
updated